packages/amplify-e2e-core/src/categories/auth.ts

```diff
@@ -540,6 +545,62 @@ export function addAuthWithSignInSignOutUrl(cwd: string, settings: any): Promise
   });
 }
 
+export function addAuthWithDefaultSocial_v30(cwd: string, settings: any): Promise<void> {
```

Comment:
what does the `_v30` mean in this function name?

Reply:
@ammarkarachi Do you remember making this function last time we merged Sign in with Apple? I think you added it to help pass pipeline tests.

Reply:
It should be v_4_30. I'll update